octoprint_mrbeam/static/js/loadingoverlay_viewmodel.js

```diff
@@ -13,6 +14,7 @@ $(function() {
             if (!event.target.activeElement.href) {
```

Comment:
Line 12: Please do not use `window.onbeforeunload = function` use `$( window ).on('beforeunload', function(){`

Reply:
changed!